src/OrbitGl/CaptureWindow.cpp

```diff
@@ -152,7 +154,10 @@ void CaptureWindow::LeftDown(int x, int y) {
   click_was_drag_ = false;
 
   if (time_graph_ == nullptr) return;
-  ref_time_click_ = time_graph_->GetTime(static_cast<double>(x) / viewport_.GetScreenWidth());
+
+  float world_x = viewport_.ScreenToWorld(Vec2i(x, y))[0];
```

Comment:
A comment independent from the PR. We should be consistent here. Maybe we should rename ScreenToWorld for GetWorldFromScreen and similar methods in the viewport?